<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckCalibrationErrorWidget</class>
 <widget class="QWidget" name="CheckCalibrationErrorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheckCalibrationErrorWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="loadCenter">
         <property name="text">
          <string>Load Tracked Center</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadRotations">
         <property name="text">
          <string>Load Rotations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Load Calibration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadTranslations">
         <property name="text">
          <string>Load Translations</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="checkError">
         <property name="text">
          <string>Check Error</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Save Error</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>161</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>Image</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Drawed</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>loadCenter</sender>
   <signal>clicked()</signal>
   <receiver>CheckCalibrationErrorWidget</receiver>
   <slot>loadCenter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadRotations</sender>
   <signal>clicked()</signal>
   <receiver>CheckCalibrationErrorWidget</receiver>
   <slot>loadRotations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadTranslations</sender>
   <signal>clicked()</signal>
   <receiver>CheckCalibrationErrorWidget</receiver>
   <slot>loadTranslations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkError</sender>
   <signal>clicked()</signal>
   <receiver>CheckCalibrationErrorWidget</receiver>
   <slot>checkError()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>CheckCalibrationErrorWidget</receiver>
   <slot>saveError()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CheckCalibrationErrorWidget</receiver>
   <slot>loadCalibration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadCenter()</slot>
  <slot>loadRotations()</slot>
  <slot>loadTranslations()</slot>
  <slot>checkError()</slot>
  <slot>saveError()</slot>
  <slot>loadCalibration()</slot>
 </slots>
</ui>
